packages/utilities/src/memoize.test.ts

```diff
@@ -98,6 +98,101 @@ describe('memoizeFunction', () => {
     expect(func('a')).toEqual('a4');
     expect(func('a')).toEqual('a4');
   });
+
+  it('updates the cache if the cached value is null', () => {
+    let returnFalseyValue = true;
+    let falseyValue: null = null;
+    let callback = (): number | null => {
+      if (returnFalseyValue) {
+        return falseyValue;
+      }
+      return 1;
+    };
+    let func = memoizeFunction(() => callback(), undefined, true /*ignoreNullOrUndefinedResult */);
+    expect(func()).toEqual(falseyValue);
+    returnFalseyValue = false;
+    expect(func()).toEqual(1);
+  });
+
+  it('updates the cache if the cached value is undefined', () => {
+    let returnFalseyValue = true;
```

Comment:
Similar comments to above: rename this to `returnUndefined` and get rid of the `falseyValue` variable. Same thing in the subsequent tests.

Reply:
Done